modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -157,6 +158,10 @@ public ClientExtensionEntry deleteClientExtensionEntry(
 		clientExtensionEntryLocalService.undeployClientExtensionEntry(
 			clientExtensionEntry);
 
+		_cetManager.deleteCET(
```

Comment:
I guess this was a bug, right? I mean this was not working before...

Reply:
What do you mean?